modules/nf-core/anndata/getsize/main.nf

```diff
@@ -0,0 +1,36 @@
+process ANNDATA_GETSIZE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/anndata:0.10.9--d13580e4b297da7c':
```

Comment:
I think this link is wrong, sent you some instructions on slack.

Reply:
thanks, replaced the url